lib/src/Base/Stat/SampleImplementation.cxx

```diff
@@ -2261,6 +2261,20 @@ SampleImplementation SampleImplementation::getMarginal(const Indices & indices)
   return marginalSample;
 }
 
+/* Get the marginal by identifiers */
+SampleImplementation SampleImplementation::getMarginal(const Description & description) const
+{
+  Indices indices;
+  for (UnsignedInteger i = 0; i < description.getSize(); ++ i)
+  {
+    const UnsignedInteger index = getDescription().find(description[i]);
```

Comment:
So if the description contains duplicate labels, the first one is used. It should be mentioned in the documentation.